### ISSUE TITLE
Imcompatible with Wagtail 3.0.1

### ISSUE THREAD

**Issue by polesello** [Frojd/wagtail-trash#14]
There is a wrong import for wagtail 3.0

In file wagtail_trash/utils.py replace lines 7-8 with

```
if wagtail.VERSION >= (3, 0):
    from wagtail.actions.move_page import MovePageAction
elif wagtail.VERSION >= (2, 16):
    from wagtail.core.actions.move_page import MovePageAction
```

**Comment by rinti** [comment]
thanks for the report, i will check this asap!

**Comment by rinti** [comment]
@polesello I've just released a new version - 0.3.0 with Wagtail 3 support. Thanks for reporting! :)
